tools/m2ctx.py

```diff
@@ -1,14 +1,43 @@
 #!/usr/bin/env python3
 
-import argparse, os, subprocess, sys
+import argparse
+import os
+import sys
+import subprocess
+import tempfile
 from pathlib import Path
 
-script_dir = os.path.dirname(os.path.realpath(__file__))
-root_dir = script_dir + "/../"
-src_dir = root_dir + "src/"
+script_dir = Path(os.path.dirname(os.path.realpath(__file__)))
+root_dir = script_dir / ".."
+src_dir = root_dir / "src"
+
+# Project-specific
+CPP_FLAGS = [
+    "-Iinclude",
+    "-Iassets",
+    "-Isrc",
```

Comment:
can you add `Ibuild` and `I.`, as well switch `Iassets` and `Isrc` to fully match with the Makefile?
`-Iinclude -Isrc -Iassets -Ibuild -I.`